src/validation.cpp

```diff
@@ -671,7 +691,7 @@ bool MemPoolAccept::PreChecks(ATMPArgs& args, Workspace& ws)
     // we have all inputs cached now, so switch back to dummy (to protect
     // against bugs where we pull more inputs from disk that miss being added
     // to coins_to_uncache)
-    m_view.SetBackend(m_dummy);
+    m_viewmempool.SetBackend(m_dummy);
```

Comment:
☝️ this is the main change 👆

Reply:
You switch `m_view.SetBackend(m_dummy)` to `m_viewmempool.SetBackend(m_dummy)` in commit _[validation] disconnect m_viewmempool from coins cache when unnecessary_.

I think perhaps in that commit you should just add `m_viewmempool.SetBackend(m_dummy)`, and then in the subsequent commit (_[validation] keep m_view->m_viewmempool connected_), you should remove `m_view.SetBackend(m_dummy)`. That keeps the commits more separated (the first is adding the m_viewmempool // CoinsTip() split, and the second is removing the m_view // m_viewmempool split).

Reply:
Good point! Done in the last push